components/EpisodesList/ListNavigation.styled.js

```diff
@@ -0,0 +1,26 @@
+import { css } from "styled-components";
+import styled from "styled-components";
+
+export const ListOptions = styled.section`
+  display: flex;
+  flex-direction: row;
```

Comment:
normaly you don't need flex-direction row ..because by default flex is row

Reply:
fixed it